main.py

```diff
@@ -208,41 +199,117 @@ def get_flows_to_be_installed(self):
                     # Change table_id before being added
                     flow["flow"].update({"table_id": expected_table_id})
                     install_flows[switch].append(flow["flow"])
+
         log.info(f"of_multi_table pushing flows, pipeline: {pipeline}")
+        self.manage_miss_flows(pipeline, flows_by_swich)
         self.send_flows(delete_flows, "delete")
         self.send_flows(install_flows, "install")
+
         if pipeline.get("status") is None:
             self.pipeline_controller.disabled_pipeline(pipeline_id)
             msg = f"Pipeline {pipeline_id} disabled"
             log.debug(f"of_multi_table result {msg}")
         else:
-            self.install_miss_flows(pipeline)
             self.pipeline_controller.enabled_pipeline(pipeline_id)
             msg = f"Pipeline {pipeline_id} enabled"
             log.debug(f"of_multi_table result {msg}")
 
-    def install_miss_flows(self, pipeline: dict):
+    @staticmethod
+    def get_miss_flows(flows_by_swich: Dict[str, Dict]):
```

Comment:
Can you also add the returned type `-> tuple[dict[str, dict], set[str]]` for completeness here?

Also, could you rename `flow_ids` to be `stored_table_ids` or something else similar that makes it clear it's a table id? `flow_ids` is storing table ids, when I was reading the code the variable name can lead to initial confusion, especially since our flow data models also have an id.

Reply:
Added the return type and change the naming.